mobile/voice_journal_app/lib/Emotions_enums.dart

```diff
@@ -0,0 +1,10 @@
+// ignore: file_names
+enum Emotions{
+  sad,
+  happy,
+  fear,
+  contempt,
```

Comment:
We aren't using 'contempt' as one of our emotions so let's remove it here

Reply:
Just noticed these aren't in the same order as we specified in the schema, let's fix that so they're the same as the backend:
```
enum Emotions {
    happiness, // id = 0
    sadness, // id = 1
    anger, // id = 2
    fear, // id = 3
    disgust, // id = 4
    surprise, // id = 5
}
```